pkg/backend/httpstate/stack.go

```diff
@@ -226,8 +235,9 @@ func (s *cloudStack) DefaultSecretManager(info *workspace.ProjectStack) (secrets
 // cloudStackSummary implements the backend.StackSummary interface, by wrapping
 // an apitype.StackSummary struct.
 type cloudStackSummary struct {
-	summary apitype.StackSummary
-	b       *cloudBackend
+	summary    apitype.StackSummary
+	b          *cloudBackend
+	defaultOrg string
```

Comment:
Worth a comment? Reading the code, it looks like our logic here is different to a stack reference -- where there we say that it could be `""`, at which point user will be defaulted, here it looks like this always gets a value, which may be the user?

Reply:
`defualtOrg` here is entirely pass-through to `cloudBackendReference`, which will still default to individual org if unset. I will add a comment!